libs/feature/notifications/tsconfig.lib.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../dist/out-tsc",
+    "target": "es2015",
```

Comment:
es2015 or es2020? not that it really matters...

Reply:
I moved all the tsconfig using es2015 (mostly the spec ones) to es2020 for uniformity.
The apps use es2022 but for some reason using es2022 everywhere breaks the tests, so I stopped at es2020. Thanks for the pointer!